### DISCUSSION TITLE
How to do a component for Multiple Checkbox using React-hook-form in react-native

### DISCUSSION THREAD

**Discussion by manongguard** [react-hook-form/react-hook-form#3856] [Q&A]
I was looking from examples from React samples but it seems im going nowhere. Anybody can help ?

**Comment by bluebill1049** [comment]
have you looked at the Controller section?

**Comment by manongguard** [comment]
@bluebill1049 i haven't yet, can you give me a link to start with? it seems I am really confused right now.

**Reply by bluebill1049** [reply]
https://react-hook-form.com/api#useController

**Comment by manongguard** [comment]
I think I got it now. Thanks!

edit: can't seem to past properly my sample code. but thanks anyway!

**Reply by mikeportanova** [reply]
Hi there,

Can you post your solution? I am having trouble figuring this out in react-native using Controller.
